packages/react-components/src/components/DevicePermissions/DomainPermissionsScaffolding.tsx

```diff
@@ -116,19 +117,25 @@ export const DomainPermissionsContainer = (props: DomainPermissionsContainerProp
   const theme = useTheme();
   return (
     <Stack style={{ padding: '2rem', maxWidth: '25.375rem' }} aria-label={strings?.ariaLabel}>
-      <Stack horizontal style={{ paddingBottom: '1rem' }} horizontalAlign={'space-between'}>
+      <Stack
+        styles={iconBannerContainerStyles}
+        horizontal
+        horizontalAlign={'center'}
```

Comment:
This can just be `horizontalAlign="center"`

https://reactjs.org/docs/introducing-jsx.html#specifying-attributes-with-jsx